Formula/swimat.rb

```diff
@@ -0,0 +1,24 @@
+class Swimat < Formula
+  desc "Command Line Tool to help format Swift code"
+  homepage "https://github.com/Jintin/Swimat"
+  url "https://github.com/Jintin/Swimat.git",
+    :tag => "v1.2.1"
+  sha256 "08876d0c13fc7782ada2cd2bbbf9724399ccea216caee0cfaf2acf298eb5b3ac"
+  head "https://github.com/Jintin/Swimat.git"
+
+  depends_on :xcode => "8.0"
+
+  def install
+    xcodebuild "-target", "CLI",
+    "-configuration", "Release",
```

Comment:
This and below lines of the `xcodebuild` should be aligned to the arguments of the first line.